docs/release-notes/release-notes-0.14.0.md

```diff
@@ -12,6 +12,26 @@ instantaneous. Read the [guide on leader
 election](https://github.com/lightningnetwork/lnd/blob/master/docs/leader_election.md)
 for more information.
 
+## Postgres database support
+
+This release adds support for Postgres as a database backend to lnd. Postgres
+has several advantages over the default bbolt backend:
+* Better handling of large data sets.
+* On-the-fly database compaction (auto vacuum).
+* Database replication.
+* Inspect data while lnd is running (bbolt opens the database exclusively).
+* Usage of industry-standard tools to manage the stored data, get performance
+  metrics, etc.
+
+Furthermore, the SQL platform opens up possibilities to dramatically improve
+lnd's performance. Bbolt's single-writer model is a severe performance
```

Comment:
nit: add `lnd's performance in the future`. Otherwise this sounds like this was already achieved (including structured tables etc).

Reply:
Definitely didn't want to suggest that. But agreed that 'opens up possibilities' can be read as if there are possibilities already now that can be utilized by the user. Changed wording.